local/bin/py/build/configurations/pull_config.yaml

```diff
@@ -1061,6 +1061,28 @@
             - link: "real_user_monitoring/explorer/"
               tag: "Documentation"
               text: "Learn how to explore your RUM data"
+    - action: pull-and-push-file
+      branch: develop
```

Comment:
Should this be "main" or "master" instead of "develop?"

Reply:
@urseberry Thanks Ursula, I'm not sure what the protocol is, but I changed it to `develop` so the link checker would pass. I was planning to change it to `main` when the source PR was merged into `main`. I was told we would get a PR for that.